docs/admin/admin-ui/scripts.md

```
@@ -25,8 +25,8 @@ The Janssen Authentication Server leverages custom scripts, which, when implemen
 
 ## Script properties
 
-Each script presents a few default properties where values can be set 
-according to the requirement. Details can be found [here](https://docs.jans.io/stable/janssen-server/developer/scripts/#configurable-properties-of-a-custom-script).
+Each script presents a few default properties where values can be set
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)